deploy/roles/python/tasks/main.yml

```diff
@@ -1,9 +1,16 @@
 # main.yml - Install Python 2.7, either from a repo or a built RPM
 ---
-- name: Install Python27 from repo
+- name: Install epel-release
   yum:
-    name: python27
+    name: ['epel-release']
+    state: latest
+ 
+  tags:
+    - python
+- name: Install Python from repo
+  yum:
+    name: ['python','python-setuptools','python-pip']
```

Comment:
We need to use `centos-release-scl` for this since just `python` means 2.6.  Installing `centos-release-scl` should probably go in the "common" role since it will be needed everywhere.